index.html

```diff
@@ -32,108 +50,117 @@ <h2>I’am a Software Developer</h2>
         </section>
         <!-- works section -->
         <section class="works-section">
-            <h1>My Recent Works</h1>
+            <h1 class="works-h1">My Recent Works</h1>
+            <div class="desktop-v line-h1-works"></div>
 
             <img src="img/work-img.png" alt="this is an image of the type of work">
 
-            <h2>Multi-Post Stories</h2>
-            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
-            <div class="tags">
-                <a><div class="tag-btn">css</div></a>
-                <a><div class="tag-btn">html</div></a>
-                <a><div class="tag-btn">bootstrap</div></a>
-                <a><div class="tag-btn">Ruby</div></a>
-            </div>
-            <div class="works__action-btn">
-                <button type="button" class="works-btn">See Project</button>
-            </div>
-
-            <div class="card-works">
-                <div class="card-content">
-                    <h2>Profesional Art Printing Data</h2>
-                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-                    <ul class="tecnologies-used">
-                        <li>html</li>
-                        <li>bootstrap</li>
-                        <li>Ruby</li>
-                    </ul>
-                    <div class="action-btn">
-                        <button type="button">See Project</button>
+            <article class="works-article">
+                <h2>Multi-Post Stories</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <div class="tags">
+                    <a><div class="tag-btn">css</div></a>
+                    <a><div class="tag-btn">html</div></a>
+                    <a><div class="tag-btn">bootstrap</div></a>
+                    <a><div class="tag-btn">Ruby</div></a>
+                </div>
+                <div class="works__action-btn">
+                    <button type="button" class="works-btn">See Project</button>
```

Comment:
- [ ] Please, consider fixing the styling for all buttons on your page. The `active` status (pressed status) for buttons is not correct. Have a look to this gif:
![2022-06-02 11 51 40](https://user-images.githubusercontent.com/49064106/171583139-f65cc447-1f06-458a-bb0d-bf08d6e8d9aa.gif)
> The original design for buttons, considering the Figma is:
<img width="364" alt="Screen Shot 2022-06-02 at 11 51 58" src="https://user-images.githubusercontent.com/49064106/171583224-c563e960-948c-480d-9fe7-7d6c2874d4dc.png">

Reply:
Implemented 👍